Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -8542,6 +8549,11 @@ - (void)_elementDidBlur
 
     [self _endEditing];
 
+#if ENABLE(CONTENT_INSET_BACKGROUND_FILL)
+    if (!_isChangingFocus)
```

Comment:
It's not immediately clear why `!_isChangingFocus` needs to be checked, can you explain?

Reply:
Ah, the `!_isChangingFocus` check is here to prevent `HideScrollPocketReason::SiteSpecificQuirk` from being removed and added again in the case where you're moving from one focused input to another (both of which would trigger this quirk).

I can clarify this in the commit message.